lib/v2/polkaworld/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'polkaworld.org': {
+        _name: 'PolkaWorld',
+        www: [
+            {
+                title: 'polkadot',
```

Comment:
title here should match your H3 header in the docs
docs: https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-radar-gui-ze-bian-xie-gui-ze